lib/elixir/lib/stream.ex

```diff
@@ -87,14 +87,14 @@ defmodule Stream do
     * `IO.stream/2`         - streams input lines, one by one
     * `URI.query_decoder/1` - decodes a query string, pair by pair
 
-  This module also provides many convenience functions for creating streams,
+  This module also provides many convenient functions for creating streams,
```

Comment:
An object such as a function can be for convenience, no?

Reply:
Yeah, but isn't the grammar different? :) you have an extra `for` in your sentence here.

Reply:
"Convenience functions" could be thought of "public helper functions"

Reply:
Thanks guys, seems like an idiom in the programming world that I wasn't aware of :) Closing this now.